cl.conf

```diff
@@ -95,20 +95,22 @@ storage:
           DOCKER_VERSION=$(docker version -f {{.Client.Version}})
           echo "Docker client version to install ${DOCKER_VERSION}"
 
-          echo "Cleaning up any existing directories"
           USER_HOME=$(echo /Users/* | head -1)
-          rm -rf ${USER_HOME}/.bricks/bin
-          mkdir -p ${USER_HOME}/.bricks/bin
-          echo "Done cleaning existing directory"
+          DOCKER=${HOME}/.docker
+          mkdir -p $DOCKER
 
           URL=https://download.docker.com/mac/static/edge/$(uname -m)/docker-${DOCKER_VERSION}.tgz
 
           echo "Fetching docker image from $URL"
+          curl -s -o - $URL | tar xzf - -O docker/docker > ${DOCKER}/docker
+
+          sudo chmod +x ${HOME}/.docker/docker
```

Comment:
maybe use ${DOCKER}/docker here for consistency

Reply:
updating